scripts/process_from_txt_to_list_tracknet.py

```diff
@@ -23,12 +23,11 @@
 FLAGS = flags.FLAGS
 
 @gin.configurable()
-def process_input_data(file_name,
+def process_input_data_to_lists(file_name,
```

Comment:
what is the purpose of this script, why not prepare.py with config?

Reply:
it's what Egor asked for earlier (may be outdated)